templates/field/field--node--oe-consultation-outcome-files--oe-consultation--full.html.twig

```diff
@@ -0,0 +1,11 @@
+{#
+/**
+ * @file
+ * Theme override for the Documents field template of the Consultation content type.
+ *
+ * @see ./core/themes/stable/templates/field/field.html.twig
+ */
+#}
+<div class="ecl-u-mt-m">
+  {%- include 'field--multiple-values.html.twig' -%}
+</div>
```

Comment:
Not sure if it is a known issue but the width doesn't corresponds to the req page.
![image](https://user-images.githubusercontent.com/22004498/105711579-0a815000-5f19-11eb-8320-452b0576244f.png)

Reply:
ECL limits to 80 characters per line which is intended. We use the file template provided by twig to render the documents. Should the template be patched or increase the 80 characters limit? We could ask Tamas about it.
<img width="1252" alt="ecl-editor p" src="https://user-images.githubusercontent.com/22908988/105774579-15b29b00-5f6e-11eb-91d7-6a81a708120f.png">

Reply:
Checked with Tamas, it's like this other places as well.